test/ess-r-tests.el

```diff
@@ -4,11 +4,17 @@
 (require 'ess-r-tests-utils)
 
 
-;;; Inferior R
+;;; R
 
 (ert-deftest ess-r-inherits-prog-mode ()
-  (with-r-file nil
-    (should (derived-mode-p 'prog-mode))))
+  (when (getenv "TRAVIS")
```

Comment:
hmm I don't think we should be testing only on Travis. How about something like

```lisp
(let ((map (current-local-map))
      found)
  (while (and map (not found))
    (if (eq (keymap-parent map) prog-mode-map)
        (setq found t)
      (setq map (keymap-parent map))))
  found)
```

This kind of test might be useful when/if we refactor ESS modes.